content/en/network_monitoring/devices/profiles.md

```diff
@@ -64,6 +64,29 @@ Profiles can be used interchangeably, meaning devices that share MIB dependencie
 
 For more Datadog provided profiles, see the [GitHub repository][3].
 
+### Metadata definition by profile
+
+Profiles have a metadata section that can be used to define where and how metadata should be collected. Values can be static or come from an OID value.
+Supported fields can be found [here][4].
+
+With Datadog Agent version 7.52 and later, a new `device_type` field is supported as a device metadata. This can be set manually in the profile and can be used to filter on specific types of devices. Accepted values include:
+
+- access_point
+- firewall
+- load_balancer
+- pdu
+- printer
+- router
+- sd-wan
+- sensor
+- server
+- storage
+- switch
+- ups
+- wlc
+
+To know more about the profiles format, see the [Profile Format Reference][5].
```

Comment:
```suggestion
See the [Profile Format Reference][5] for more information about profiles formats.
```

Reply:
adresed in [5c48a7a](https://github.com/DataDog/documentation/pull/21347/commits/5c48a7a914c0a2cc53f4c4f2498b140f9fa8c7ef)